new YAMLs TJB

tjbarnum13 opened the pull request:
new YAMLs for quinoline, pyridazine, isoquinoline, and imidazole

tjbarnum13 committed:
Create quinoline.yml

tjbarnum13 committed:
Create pyridazine.yml

A bit unsure of the hfs implementation

tjbarnum13 committed:
Create isoquinoline.yml

tjbarnum13 committed:
Create imidazole.yml

bchangala commented on data/isoquinoline.yml:
```diff
@@ -0,0 +1,21 @@
+doi: [10.1016/S0022-2852(02)00020-6]
+name: isoquinoline
+smiles: C1=CC=C2C=NC=CC2=C1
+notes: C9H7N
```
change to formula

tjbarnum13 committed:
Update isoquinoline.yml

bchangala commented on data/pyridazine.yml:
```diff
@@ -0,0 +1,32 @@
+chi_aa_2: -4.6411
+chi_bb_2: 1.3711
+chi_cc_2: 3.2700
+chi_ab_2: 2.709
```
I think the off-diagonal chi_ab constants for 1 and 2 should have signs opposite of each other, but which one is + and which one is - is not determined.

tjbarnum13 replied:
great - that sounds reasonable to me. i knew you would have the answer bryan!

bchangala reviewed (changes requested):
See note on sign of pyridazine chi's

tjbarnum13 committed:
Update pyridazine.yml

Make off-diagonal chi_ab opposite signs for nitrogen 1 and 2. (which one is which not determined in fit)
